### ISSUE TITLE
Consumes memory usage grows until computer (Win 10) is unusable

### ISSUE THREAD

**Issue by dennismf** [mbunkus/mkvtoolnix#1631]
I installed version 8.9.0 (64 bit) on my Windows 10 machine after many years using a much older version of your software. When I start mkvtoolnix_gui.exe the program starts consuming memory. It quickly consumes all that remains of the 8gb of physical memory. The virtual mamual usage continues to grow, I can usually kill it by the time it reaches an image size of around 22gb.

I also tried the 32 bit version. It appears to fail very quickly, possibly because it quickly reaches the 32 bit memory limit. I could not find any messages in the event logs.

I eventually removed 8.9.0 and installed 8.8.0 (64 bit). This version is working quite well.

Thank you for your valuable program and your contribution to the video processing community.

Dennis Mackay-Fisher

**Comment by mbunkus** [comment]
Thanks for the report. This doesn't happen normally; in fact a new 8.9.0 installation on my Win10 only consumes about 10.5 MB of memory when running.

Can you please…

1. uninstall 8.8.0 again,
2. remove all traces of stored settings and job files (see [this FAQ entry](/mbunkus/mkvtoolnix/wiki/Location-of-settings-and-the-job-queue) for where they're stored),
3. re-install 8.9.0 64bit and try launching it again?

Thanks.

**Comment by stevek123** [comment]
I have exactly the same issue since 8.9.0. Deleting the job files solves the issue, but it comes back, I always have to delete them.

**Comment by mbunkus** [comment]
What do you mean, you "always have to delete them"? How much memory does the GUI consume if you do NOT remove the jobs before starting the GUI? How big is the job directory in that moment?

**Comment by stevek123** [comment]
Right now, I only have 1 job in the job directory (630 KB), and after I start the MKVToolNix GUI, it uses more than 2GB of RAM.

**Comment by mbunkus** [comment]
Thanks for the information. I still cannot reproduce it, even with a job queue directory > 1MB in size.. Can you send me that job file to moritz@bunkus.org, please?

Also: have you done anything else except for starting the GUI before looking at its memory usage (e.g. run a job, added a file etc.)?

**Comment by stevek123** [comment]
No, I haven't. One time there was like 15-20 jobs in that folder, and the program started running, but the GUI never showed up, and the whole computer started lagging, and the RAM usage was just growing and growing... I was barely able to kill the process from the task manager. I have sent you the job file.

**Comment by mbunkus** [comment]
Thank you for the file. I can reproduce it with it and will investigate why it happens.

**Comment by mbunkus** [comment]
The pre-builds are up. I'd appreciate some testing.

**Comment by stevek123** [comment]
It works for me. I will let you know if the issue occurs again. Thanks for the fix
